addons/mail/static/src/models/welcome_view/welcome_view.js

```diff
@@ -41,6 +41,10 @@ function factory(dependencies) {
             }
             if (this.discussPublicView.shouldAddGuestAsMemberOnJoin) {
                 await this.performRpcAddGuestAsMember();
+                // We restart the user polling to take the new channel into
```

Comment:
We usually avoid 1st person in comments, just "Restart" without "we" would be enough here